src/Generator/PropertyGenerator.php

```diff
@@ -136,40 +148,62 @@ public function __construct($name = null, $defaultValue = null, $flags = self::F
         }
     }
 
-    /**
-     * @param  bool $const
-     * @return PropertyGenerator
-     */
-    public function setConst($const)
+    public function setConst(bool $const): self
```

Comment:
`:self` cannot be added here.

`@return static` should probably be kept at docblock level?

Reply:
Reverted all signature changes but kept `@return static`